ReactQt/runtime/src/rootview.cpp

```diff
@@ -112,96 +110,100 @@ private Q_SLOTS:
     }
 };
 
-ReactView::ReactView(QQuickItem* parent) : ReactItem(parent), d_ptr(new ReactViewPrivate(this)) {
-    Q_D(ReactView);
+RootView::RootView(QQuickItem* parent) : ReactItem(parent), d_ptr(new RootViewPrivate(this)) {
+    Q_D(RootView);
 
     setAcceptedMouseButtons(Qt::LeftButton);
     setFiltersChildMouseEvents(true);
 
     d->bridge = new ReactBridge(this);
     connect(d->bridge, SIGNAL(readyChanged()), SLOT(bridgeReady()));
 
-    connect(this, SIGNAL(liveReloadChanged()), d, SLOT(onLiveReloadChanged()));
+    connect(this, SIGNAL(liveReloadChanged()), d, SLOT(liveReloadChanged()));
```

Comment:
I'm getting warning on runtime
`QObject::connect: No such slot RootViewPrivate::liveReloadChanged() in ..src/rootview.cpp:122`

Reply:
Thanks for noticing. Incorrectly merged.